index.js

```diff
@@ -1,16 +1,18 @@
-"use strict"
-var os = require("os")
+"use strict";
+const os = require("os");
 
-var hasUnicode = module.exports = function () {
+const hasUnicode = module.exports = function () {
   // Recent Win32 platforms (>XP) CAN support unicode in the console but
   // don't have to, and in non-english locales often use traditional local
   // code pages. There's no way, short of windows system calls or execing
   // the chcp command line program to figure this out. As such, we default
   // this to false and encourage your users to override it via config if
   // appropriate.
-  if (os.type() == "Windows_NT") { return false }
+  if (os.type() === "Windows_NT") {
```

Comment:
More properly, this line should be `if (os.type() === "Windows_NT") return false` (that is, I would just get rid of the pointless block, rather than making it multiline.)